### ISSUE TITLE
WinDirStat 2.0.3 bug

### ISSUE THREAD

**Issue by RandyInHawaii** [windirstat/windirstat#133]
In the graph area of the results, when hovering the mouse over a block, the file name only appears briefly in the status bar, then disappears and the word Ready appears instead.  This is not how the old version behaved, nor do I suspect it is behaving as intended.  I have verified the issue on several PC's ranging from Win 10, 11 and server versions.

**Comment by NoMoreFood** [comment]
Yes, this has been fixed in the beta. Should be released in the next week or two.

**Comment by NoMoreFood** [comment]
Released with https://github.com/windirstat/windirstat/releases/tag/release/v2.1.1
